tests/conftest.py

```diff
@@ -956,7 +956,7 @@ def cluster_teardown_finalizer():
         deployer.deploy_cluster(log_cli_level)
 
 
-@pytest.fixture(scope='class')
+@pytest.fixture()
```

Comment:
Not sure if its good idea to change this now, it will require all tier testing. Also, we want the scope to be class since we will have dependency across test within the same class, so some test may fail unnecessarily.

Reply:
realized this after the job verification - thanks.
Fixed